### PR TITLE
REL-2720: More adjustments to position angle field

### PR DESCRIPTION
**Pull request by sraaphorst**
As BAGS can use the "Flip 180" position angle feature in order to calculate a position angle +180 of the position angle currently being typed into the position angle field, we have special handling in place as per an earlier PR for this feature to prevent the contents of the PA field from being changed until the focus is gained / lost.

Andy reopened the JIRA task stating that this change should also occur when ENTER is pressed in the field, so this small PR adds that functionality.

### CODE REVIEW

File: bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/EdCompInstBase.java [hunk 1]
```diff
@@ -91,13 +100,14 @@ public void textBoxKeyPress(final TextBoxWidget tbwe) {
...
-    private void updatePosAngle() {
+    private void updatePosAngle(final boolean ignoreFocus) {
         if (!_ignoreChanges) {
             // Ignore model changes to the pos angle if the pos angle text box has the focus.
             // This is to avoid changing the text box value when BAGS selects an auto group at +180.
             final TextBoxWidget posAngleTextBox = getPosAngleTextBox();
-            if (posAngleTextBox != null && getDataObject() != null && !posAngleTextBox.hasFocus()) {
+            if (posAngleTextBox != null && getDataObject() != null && (ignoreFocus || !posAngleTextBox.hasFocus())) {
                 final String newAngle = getDataObject().getPosAngleDegreesStr();
+                System.out.println("--- newAngle=" + newAngle);
```
Comment (cquiroz) [inline bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/EdCompInstBase.java]: Do we need the sysout?

**Comment by cquiroz** [comment]
Looks good to me :+1:

File: bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/EdCompInstBase.java (hunk 1 above)
Comment (tpolecat) [inline bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/EdCompInstBase.java]: Yeah please remove this or put it in a low-priority log message.

**Comment by sraaphorst** [comment]
Removed, and sorry. Ugh. Bad habit of mine.

**Pull request merged**